src/screens/SendFunds/03-Amount.js

```diff
@@ -311,27 +355,39 @@ class SendAmount extends Component<Props, State> {
             <TouchableWithoutFeedback onPress={this.blur}>
               <View style={{ flex: 1 }}>
                 <AmountInput
+                  editable={!useAllAmount}
                   account={account}
                   onChange={this.onChange}
                   currency={unit.code}
-                  value={amount}
+                  value={useAllAmount ? maxAmount : amount}
                   error={inlinedError}
                 />
 
                 <View style={styles.bottomWrapper}>
-                  <LText style={styles.available}>
-                    <Trans i18nKey="send.amount.available">
-                      {"text"}
+                  <View style={styles.available}>
+                    <View style={styles.availableLeft}>
+                      <LText>
+                        <Trans i18nKey="send.amount.available" />
+                      </LText>
                       <LText tertiary style={styles.availableAmount}>
                         <CurrencyUnitValue
                           showCode
                           unit={unit}
                           value={account.balance}
                         />
                       </LText>
-                      {"text"}
-                    </Trans>
-                  </LText>
+                    </View>
+                    <View style={styles.availableRight}>
+                      <LText style={styles.maxLabel}>
+                        <Trans i18nKey="send.amount.useMax" />
+                      </LText>
+                      <Switch
+                        style={{ opacity: 0.99 }}
```

Comment:
why? if there is a reason, please comment a `// FIXME` explanation and/or link (typically we later can check for these fixme comment)

Reply:
Any animation involving opacity makes some buttons like radio buttons and switches flash on some versions of Android, this was the solution I ranted about at https://github.com/LedgerHQ/ledger-live-mobile/issues/589 don't know if that's detailed enough for you, I've been using that ever since. If you think there's a need to document this behaviour further, I can add a reference to the link above each time we use it